src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -49,13 +49,6 @@ public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBas
         public List<AzureSqlDatabaseModel> Database { get; set; }
 
         /// <summary>
-        /// Gets or sets the tags associated with the Azure SQL Database Failover Group
```

Comment:
Yes. I have made the changes to revert breaking changes and mark them as deprecated. Please review and let me know if it looks good.

Thanks,
Mike